helm-chart/templates/nextcloud-aio-talk-service.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if eq .Values.TALK_ENABLED "yes" }}
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    kompose.cmd: kompose convert -c -f latest.yml
+    kompose.version: 1.27.0 (b0ed6a2c9)
+  labels:
+    io.kompose.service: nextcloud-aio-talk
+  name: nextcloud-aio-talk
+spec:
+  ports:
+    - name: "{{ .Values.TALK_PORT }}"
+      port: {{ .Values.TALK_PORT }}
+      targetPort: {{ .Values.TALK_PORT }}
```

Comment:
Also here better set it to TCP explicitly

Reply:
thanks for the suggestion but I think this is fine